themes/gauge_theme/static/css/theme.css

```diff
@@ -700,7 +691,7 @@ header.top {
   content: "";
   display: block; }
 @media (max-width: 767.98px) {
-  nav.nav {
+  .nav {
```

Comment:
you need to use `nav.nav`, otherwise the style break the codetabs which also use `nav` class.